backend/utils/authUtils.js

```diff
@@ -0,0 +1,28 @@
+import uuidv4 from 'uuid/v4';
+import {saveSetting} from '../settings.js';
+
+const ACCESS_TOKEN_EXPIRY = 300;
+
+export function generateAndSaveAccessToken(){
+    const access_token = uuidv4();
+    saveSetting('ACCESS_TOKEN', access_token);
+    return access_token;
+}
+
+
+export function isElectron() {
+
+    if (process.versions && !!process.versions.electron) {
```

Comment:
🐮  instead of `!!`, I prefer something more explicit like `R.has('electron', process.versions)` (in general, I try to use `ramda` wherever possible - it's like our "standard library")